ui/packages/consul-ui/app/styles/base/color/README.mdx

```diff
@@ -0,0 +1,81 @@
+# Colors
+
+All colors for the immediate future should _mostly_ use the following `tone`
+CSS properties, unless:
+
+1. We get around to eventually adding semantic colors names (or adding something else) in which case these docs should be updated.
+2. You are absolutely sure the color you want should never be changed by a theme. These should be very very few and far between so even if we get that wrong it should be easy to update.
+
+`tones` should always using the following form:
+
+```css
+property: rgb(var(--tone-gray-500));
+```
+
+In other words always use `rgb` this gives us the flexibility to easily use thee variables with alpha if we ever need to.
+
+```css
+property: rgb(var(--tone-gray-500) / 50);
+```
+
+Currently, alphas are mainly used for shadows. Please consider whether you should be using an alpha before using an alpha'ed color.
+
+Lastly, there is currently one non-color alias group which is `--tone-brand` which is a group of our current brand color.
+
+
+## %theme-light (default)
+
+```hbs preview-template
+<ul
+  {{css-props (set this 'tones')
```

Comment:
Ah ha, so this is what that modifier does. Cool 😎

Reply:
😸 well kinda, tbh this one started out as a "only give me the variables for this element" - the important bit being "for this element". This one turned into getting things from the root element (not the modified element) so 'theoretically' shouldn't be a modifier - in the scheme of things (and as its only for documentation purposes) I just left it as is. I may move it to be a helper at some point but no biggie really (unless we use it in app code)

I always decide on a modifier if ever the "I just need a reference to the DOM element" slips into my head. The below `with-copyable` is probably a better example.